src/App.js

```diff
@@ -10,4 +10,4 @@ const App = ()=> {
     )
 }
 
-export default App
+export { App };
```

Comment:
Why change from default export to named export? You are breaking other people's codes.

Reply:
Resolved. The issue was related to the use of incompatible Adapter